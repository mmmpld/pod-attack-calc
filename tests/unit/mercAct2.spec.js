import breakpoints from './breakpoints.js'
const characterId = 8;

describe('Merc Act2 default values', () => {
    it('jab attack no weapon', async () => {
        await breakpoints('7', '3.12', { charactersSelected: characterId });
    })

    it('jab attack thresher', async () => {
        await breakpoints('6.5', '3.33', { charactersSelected: characterId, weaponsPrimarySelected: 248 });
    })

    it('jab attack mancatcher', async () => {
        await breakpoints('6', '3.57', { charactersSelected: characterId, weaponsPrimarySelected: 172 });
    })
})