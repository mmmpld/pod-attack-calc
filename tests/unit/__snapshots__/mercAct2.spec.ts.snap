// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Merc Act2 default values > jab attack mancatcher 1`] = `
"<tbody>
  <tr class=\\"highlight-current-row\\">
    <th>0</th>
    <td>6</td>
    <td>3.57</td>
  </tr>
  <tr class=\\"\\">
    <th>9</th>
    <td>5.5</td>
    <td>3.84</td>
  </tr>
  <tr class=\\"\\">
    <th>26</th>
    <td>5</td>
    <td>4.16</td>
  </tr>
  <tr class=\\"\\">
    <th>52</th>
    <td>4.5</td>
    <td>4.54</td>
  </tr>
  <tr class=\\"\\">
    <th>102</th>
    <td>4</td>
    <td>5</td>
  </tr>
</tbody>"
`;

exports[`Merc Act2 default values > jab attack no weapon 1`] = `
"<tbody>
  <tr class=\\"highlight-current-row\\">
    <th>0</th>
    <td>7</td>
    <td>3.12</td>
  </tr>
  <tr class=\\"\\">
    <th>9</th>
    <td>6.5</td>
    <td>3.33</td>
  </tr>
  <tr class=\\"\\">
    <th>20</th>
    <td>6</td>
    <td>3.57</td>
  </tr>
  <tr class=\\"\\">
    <th>37</th>
    <td>5.5</td>
    <td>3.84</td>
  </tr>
  <tr class=\\"\\">
    <th>63</th>
    <td>5</td>
    <td>4.16</td>
  </tr>
  <tr class=\\"\\">
    <th>105</th>
    <td>4.5</td>
    <td>4.54</td>
  </tr>
  <tr class=\\"\\">
    <th>200</th>
    <td>4</td>
    <td>5</td>
  </tr>
</tbody>"
`;

exports[`Merc Act2 default values > jab attack thresher 1`] = `
"<tbody>
  <tr class=\\"highlight-current-row\\">
    <th>0</th>
    <td>6.5</td>
    <td>3.33</td>
  </tr>
  <tr class=\\"\\">
    <th>8</th>
    <td>6</td>
    <td>3.57</td>
  </tr>
  <tr class=\\"\\">
    <th>22</th>
    <td>5.5</td>
    <td>3.84</td>
  </tr>
  <tr class=\\"\\">
    <th>42</th>
    <td>5</td>
    <td>4.16</td>
  </tr>
  <tr class=\\"\\">
    <th>75</th>
    <td>4.5</td>
    <td>4.54</td>
  </tr>
  <tr class=\\"\\">
    <th>142</th>
    <td>4</td>
    <td>5</td>
  </tr>
</tbody>"
`;
